packages/core/src/connectivity/scp-cf/destination/destination-cache.spec.ts

```diff
@@ -9,7 +9,7 @@ import {
   subscriberUserJwt
 } from '../../../../test/test-util/mocked-access-tokens';
 import {
-  mockedConnectivityServiceProxyConfig,
+  connectivityProxyConfigMock,
```

Comment:
This is a test comment using the new webStorm PR review tool.

Reply:
Worx ;)